lib/standard/file.nit

```diff
@@ -517,12 +550,15 @@ class FileStat
 	end
 
 	# Is this a regular file and not a device file, pipe, socket, etc.?
-	fun is_file: Bool
+	fun is_reg: Bool
 	do
 		assert not finalized
 		return stat.is_reg
 	end
 
+	# Alias for `is_reg`
+	fun is_file: Bool do return is_reg
```

Comment:
I can alias `is_reg` if you prefer. for some reason I do not know the name
is not the same in NativeFileStat and in FileStat so in order to not break
to much things, I created an alias.

The comment is `Is this a regular file?` what is important seems the
regular, not the file.
One could reuse to comment from `test(1)` that says `FILE exists and is a
regular file` that suits more to `is_file`



On Tue, Apr 7, 2015 at 9:00 PM, Alexandre Terrasa <notifications@github.com>
wrote:

> In lib/standard/file.nit
> <https://github.com/privat/nit/pull/1246#discussion_r27879737>:
>
> >  	do
> >  		assert not finalized
> >  		return stat.is_reg
> >  	end
> >
> > +	# Alias for `is_reg`
> > +	fun is_file: Bool do return is_reg
>
> Why are you aliasing it? is_file is a pretty common name for that
> behavior.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/privat/nit/pull/1246/files#r27879737>.
>